model/lib/email/presentation_email.dart

```diff
@@ -135,6 +142,22 @@ class PresentationEmail with EquatableMixin, SearchSnippetMixin {
 
   bool get hasCalendarEvent => headerCalendarEvent?[IndividualHeaderIdentifier.headerCalendarEvent]?.isNotEmpty == true;
 
+  bool get isMarkAsImportant {
+    final xPriority = xPriorityHeader?[IndividualHeaderIdentifier.xPriorityHeader]
+      ?.trim()
+      .toLowerCase();
+    final importance = importanceHeader?[IndividualHeaderIdentifier.importanceHeader]
+      ?.trim()
+      .toLowerCase();
+    final priority = priorityHeader?[IndividualHeaderIdentifier.priorityHeader]
+      ?.trim()
+      .toLowerCase();
+
+    return xPriority == MailPriorityHeader.firstXPriority &&
+      importance == MailPriorityHeader.highImportance &&
+      priority == MailPriorityHeader.urgentPriority;
```

Comment:
```suggestion
    return xPriority == MailPriorityHeader.firstXPriority ||
      importance == MailPriorityHeader.highImportance ||
      priority == MailPriorityHeader.urgentPriority;
```

Reply:
Where did you read that you should use `or`?. As far as I know it should be `and`

Reply:
Updated